lib/elixir/lib/kernel.ex

```diff
@@ -1126,6 +1126,17 @@ defmodule Kernel do
     :erlang.spawn_monitor(module, fun, args)
   end
 
+  @doc """
+  Applies a function over a term and returns term
+
+  Useful for running side effects
+  """
+  @spec tap(x, (x -> any)) :: x when x: term
```

Comment:
Use “var” instead of any. Also please add examples, @doc since, and tests.

Reply:
Also please use “value” instead of “term” and add periods at the end of sentences.

Reply:
Sorry, @josevalim, I opened the PR before finishing just to signalize work was being done